src/Page/Auth.elm

```diff
@@ -0,0 +1,698 @@
+                session
+                    |> Session.notifyError "Erreur lors du login" ""
+
+              else
+                session
+                    |> Session.logout
+            , Cmd.none
+            )
+
+        LoggedIn (Ok newProcessesJson) ->
+            let
+                newSession =
+                    Session.loggedIn session newProcessesJson
+            in
+            ( model
+            , newSession
+            , newSession.store |> Session.serializeStore |> Ports.saveStore
+            )
+
+        LoggedIn (Err error) ->
+            let
+                newSession =
+                    session
+                        |> Session.notifyError "Impossible de charger les impacts lors de la connexion" error
+            in
+            ( model
+            , newSession
+            , Cmd.none
+            )
+
+        LoggedOut ->
+            ( { model | response = Nothing }
+            , session
+            , Cmd.none
+            )
+
+        Logout ->
+            let
+                newSession =
+                    Session.logout session
+                        |> Session.notifyInfo "Vous n'avez plus accès au détail des impacts" ""
+            in
+            ( model
+            , newSession
+            , Cmd.batch
+                [ newSession.store |> Session.serializeStore |> Ports.saveStore
+                , logout
+                ]
+            )
+
+        TokenEmailSent response ->
+            ( { model | response = Result.toMaybe response }
+            , case response of
+                Ok _ ->
+                    session
+
+                Err _ ->
+                    session
+                        |> Session.notifyError "Erreur lors de la connexion" ""
+            , Cmd.none
+            )
+
+        UpdateForm newModel ->
+            ( newModel
+            , session
+            , Cmd.none
+            )
+
+
+view : Session -> Model -> ( String, List (Html Msg) )
+view session model =
+    ( "Authentification"
+    , [ Container.centered [ class "pb-5" ]
+            [ h1 [ class "mb-3" ]
+                [ text <|
+                    if Session.isAuthenticated session then
+                        "Compte"
+
+                    else
+                        "Connexion / Inscription"
+                ]
+            , div [ class "row justify-content-center" ]
+                [ if Session.isAuthenticated session then
+                    div [ class "row d-flex justify-content-center" ]
+                        [ if model.loggedIn then
+                            p [ class "text-center" ]
+                                [ text "Vous avez maintenant accès au détail des impacts, à utiliser conformément aux "
+                                , a [ href Env.gitbookUrl ] [ text "conditions" ]
+                                , text "."
+                                ]
+
+                          else
+                            text ""
```

Comment:
I think here we're gonna want to render a text explaining why the registration is needed. I don't think it's a good idea to land this onto production without it.